### ISSUE TITLE
[FP] `Static lambdas binding` triggered on nested lambdas with rebound inner lambda

### ISSUE THREAD

**Issue by jdreesen** [kalessil/phpinspectionsea#1466]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate) |
| Language level | PHP 7.2                        |

Current behaviour
--
This may be a very specific case, but when I have two nested lambdas where the outer one is static and the inner one is re-bound to another object (to change the scope of `$this`), the `Static lambdas binding` inspection raises the message `[EA] '$this' can not be used in static closures.` but it shouldn't. 

See example: https://3v4l.org/G2Md3

```php
$getProperty = static function (object $object, string $propertyName) {
    return (function (string $propertyName) {
        return $this->$propertyName;
    })->bindTo($object, $object)($propertyName);
};
```

![grafik](https://user-images.githubusercontent.com/424602/70805910-5f681280-1dba-11ea-9d6b-4a7bc505b9f3.png)

Expected behaviour
--
The inspection doesn't get triggered.

**Comment by kalessil** [comment]
Fixed!
